### DISCUSSION TITLE
Poor performance with very low-end hardware

### DISCUSSION THREAD

**Discussion by Lowtihdaflow** [open-goal/jak-project#2981] [Jak 1]
The game's frame rate is very slow and textures sometimes flicker also a tutorial on what buttons to press on pc would be nice also daxter in the intro of jak 1 daxter is invisible for a short period of time also I cannot collect any eco or even scout flies my pc specs are AMD A6-7310 APU with AMD Radeon R4 Graphics 2.00 GHz for a game made in 2001 it should not have been running like this I hope you can fix this.....

**Comment by Zedb0T** [comment]
Can you attach logs and supply us with any information?

What OS are you on? 
What version of OG are you on? etc

If you can attach a video of the issues that would also be very helpful

**Comment by xTVaser** [comment]
> a game made in 2001 it should not have been running like this

There is a lot more that goes on in making something performant than just waiting for time to pass.  Without getting into too much detail, just because hardware is newer does not mean it can do all the same things that the original PS2 hardware did.

We also make some reasonable assumptions during our development that users will have moderately decent and up to date hardware (up-to-date meaning, something that was considered good in the past 10 years).

Your GPU is extremely underpowered https://www.videocardbenchmark.net/gpu.php?gpu=Radeon+R4&id=2902 only marginally better than some GPUs that were still in popular use when the original PS2 was still relevant.  So you're really not making a fair comparison.

**Comment by water111** [comment]
Your GPU is very slow and probably can't play this game. Yes, the game was made in 2001, but with OpenGOAL the resolution, level of detail, and texture filtering are all improved over the original. These improvements mean that the game is a lot more demanding and there is not really much we can do about it.  If you don't believe us you can try the upscaling feature of PCSX2 and it will probably have similar or worse performance as opengoal.

In some aspects your GPU is barely faster than the PS2's gpu. The PS2 had a fill rate of 2.4 gigapixels/s and your GPU is 3.2. But you are probably trying to play at resolution with 5-10x the pixels as the ps2, and with 5-10x more triangles drawn on each frame.
